### ISSUE TITLE
DIPY Free-Water Corrected DTI (FA output quality)

### ISSUE THREAD

**Issue by LaraFernandez44** [dipy/dipy#2682]
## Description
Hi, I have been using the DIPY free-water corrected DTI function, fwdtimodel.fit() on multi-shell DWI data using b-values [200, 500, 1000]. It runs fine, but I have some concerns about the quality of the output of the free-water corrected FA images. They are quite 'gappy', showing small holes in corpus callosum and outer portions of the cerebellum (see attached image, sagittal view). There is also a reasonable amount of noise in the free-water spaces. Are there any parameters that I can set either at the command line, or within the source code itself that may reduce these artefacts? Would it help to specify the big and little deltas directly when instantiating the gradient table?
![DIPY_FWDTI_FA](https://user-images.githubusercontent.com/119274213/204191448-172fa228-1da9-4b69-9b3b-b91e30a90944.png)

We are using the most current version of DIPY and dependencies via Python 3.10.6 run on a Linux platform (bash, MASSIVE cluster).

Many thanks,
Lara.

**Comment by skoudoro** [comment]
Hi @LaraFernandez44,

Could you also precise all the pre-processing steps you are applying before the reconstruction with fwdti

Thank you

**Comment by LaraFernandez44** [comment]
Hi Serge,
  We used qsiprep to apply all corrections: Preprocessing — qsiprep version
documentation <https://qsiprep.readthedocs.io/en/latest/preprocessing.html>

Many thanks,
Lara.

**Comment by ShreyasFadnavis** [comment]
Hi @LaraFernandez44 ! Thank you for your question -- I think the issue is that something went wrong in the pre-processing. Can you tell us what methods did you use from QSIPrep?
